os/arch/arm/Kconfig

```diff
@@ -288,15 +288,34 @@ config ARCH_DABORTSTACK
 		will be used during data abort handling. Recommended data abort stack
 		size is 1K.
 
+
+menu "Stack Protection Feature Support"
+
+config STACK_PROTECTION_SUPPORT
```

Comment:
I think we had better add new config in choice.
```
choice

config REG

config MPU

config NO

endchoice
```
If we do that like above, all of option can be controlled by one step.
But if we keep current change, we should do 2 steps to turn on.

Reply:
done